cluster/common.sh

```diff
@@ -20,12 +20,17 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(cd $(dirname "${BASH_SOURCE}")/.. && pwd)
-
 DEFAULT_KUBECONFIG="${HOME}/.kube/config"
 
-source "${KUBE_ROOT}/cluster/lib/util.sh"
-source "${KUBE_ROOT}/cluster/lib/logging.sh"
+if [ -f "${KUBE_ROOT}/cluster/lib/util.sh" ]; then
```

Comment:
I don't think this is necessary. It seems more likely that `KUBE_ROOT` is being set incorrectly somewhere.